tools/run_tests.sh

```diff
@@ -70,6 +70,9 @@ find lib -iname "*.pyc" | xargs rm
 echo 'Cleaning old authentication tokens'
 rm -rf /tmp/.rucio_*/
 
+echo 'Cleaning RSEs'
```

Comment:
I added this because I had some problems with uploading files and restarting the tests as they are not in the database anymore.

Reply:
that's ok. but i think to be super-safe you could restrict it to

`rm -rf /tmp/rucio_rse/*` ?